packages/renderer/src/lib/extensions/ExtensionDetailsReadme.spec.ts

```diff
@@ -40,16 +33,19 @@ beforeEach(() => {
 
 test('Expect to have readme with URI', async () => {
   const spyFetch = vi.spyOn(window, 'fetch');
-  spyFetch.mockResolvedValueOnce(new Response('#'));
   spyFetch.mockResolvedValueOnce(new Response('# This is my README'));
 
   await waitRender({ readme: { uri: 'http://my-fake-registry/readme-content' } });
 
   // expect Markdown
-  const markdownContent = screen.queryByRole('region', { name: 'markdown-content' });
-  expect(markdownContent).toBeInTheDocument();
-
-  expect(markdownContent).toContainHTML('<h1 id="this-is-my-readme">This is my README</h1>');
+  await vi.waitFor(() => expect(screen.queryByRole('region', { name: 'markdown-content' })).toBeInTheDocument());
+
+  await vi.waitFor(() => expect(spyFetch).toHaveBeenCalled());
```

Comment:
is waitFor required here ?

Reply:
it's to wait that we call the fetch method to grab the readme file

Reply:
it might not be "required" but it allows to move the line one step earlier as well